packages/sidenav/src/SideNav.story.js

```diff
@@ -0,0 +1,437 @@
+import React, { useState } from "react";
+import styled from "styled-components";
+import { colors } from "@crave/farmblocks-theme";
+import Text from "@crave/farmblocks-text";
+import { Switch, Route, NavLink, withRouter } from "react-router-dom";
+import StoryRouter from "storybook-react-router";
+import { useMediaQuery } from "react-responsive";
+
+import SideNav from "./SideNav";
+import variants, { FULLSCREEN, PUSH, OVERLAY } from "./constants/variants";
+import NavItem from "./components/NavItem";
+import PageWrapper from "./helpers/PageWrapper";
+
+const tabs = ["purveyor", "order", "search", "meat"];
+
+const useToggle = initialState => {
+  const [expanded, setExpanded] = useState(initialState);
+  const toggle = () => setExpanded(e => !e);
+  const collapse = () => setExpanded(false);
+  const expand = () => setExpanded(true);
+
+  return [expanded, { toggle, collapse, expand }];
+};
```

Comment:
This is a good hook, would it be useful on webapp? If so, we should export it and use it there.
Only change I would make is to export a full object:
`{ expanded, toggle, collapse, expand }`